src/cross_section.F90

```diff
@@ -248,6 +267,12 @@ subroutine calculate_nuclide_xs(i_nuclide, i_sab, E, i_mat, i_nuc_mat, u)
       micro_xs(i_nuclide) % kappa_fission = &
            nuc % reactions(nuc % index_fission(1)) % Q_value * &
            micro_xs(i_nuclide) % fission
+
+      ! Calculate microscopic nuclide delayed nu-fission cross section
+      micro_xs(i_nuclide) % delayed_nu_fission = (ONE - f) * &
+           nuc % delayed_nu_fission(i_grid) + f * &
+           nuc % delayed_nu_fission(i_grid+1)
```

Comment:
The problem with that model is that it only includes 3 nuclides, so it's not surprising there is not much effect because there aren't very many cross section lookups to start with. I use the [Monte Carlo performance benchmark](https://github.com/mit-crpg/benchmarks/tree/master/mc-performance/openmc) (also known as Hoogenboom-Martin) often as a quick check (which has ~40-50 nuclides), but BEAVRS is a more appropriate real problem, if you have the model.